engine/engine-script-library

```diff
@@ -385,33 +382,27 @@ function get_data() {
     cs_start_bench_cmds="$cs_label-start-bench.cmds"
     # Everyone gets the bench-start-cmds (the client's start cmds) because that tells everyone how many tests there are
     if [ "$cs_type" == "client" -o "$cs_type" == "server" ]; then
-        scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/start" "bench-start-cmds"
+        scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/start" "bench-start-cmds" abort get-data-end
     else
         # All tool-only osruntimes just copy from client-1
-        scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/client/1/start" "bench-start-cmds"
-    fi
-    if [ ! -e "bench-start-cmds" ]; then
-        abort_error "bench cmds file bench-start-cmds not found. PWD: `/bin/pwd`  LS: `/bin/ls`" get-data-end
+        scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/client/1/start" "bench-start-cmds" abort get-data-end
     fi
     if [ "$cs_type" == "client" -o "$cs_type" == "server" ]; then
         if [ "$cs_type" == "client" ]; then
-            scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/infra" "bench-infra-cmds"
+            scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/infra" "bench-infra-cmds" abort get-data-end
             if [ "$cs_id" == "1" ]; then
-                scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/runtime" "bench-runtime-cmds"
+                scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/runtime" "bench-runtime-cmds" abort get-data-end
             fi
         fi
         if [ "$cs_type" == "server" ]; then
-            scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/stop" "bench-stop-cmds"
-            if [ ! -e "bench-stop-cmds" ]; then
-                abort_error "bench cmds file bench-stop-cmds not found. PWD: `/bin/pwd`  LS: `/bin/ls`" get-data-end
-            fi
+            scp_from_controller "$ssh_id_file" "$engine_config_dir/bench-cmds/$cs_type/$cs_id/stop" "bench-stop-cmds" abort get-data-end
```

Comment:
Do you really want to remove the next [ -e "bench-stop-cmds ] code block?

Reply:
The error handler that is being added to `scp_from_controller` handles this.